README.md

```diff
@@ -35,6 +35,7 @@ If you are interested in GraphQL in general, check out [awesome-graphql](https:/
 | melodyRepo | Fast and reliable dependency manager for Go | [Try it!](https://melody.sh/play/) | [Docs](https://melody.sh/docs/api)
 | React Finland | React Finland API is built with conferences and meetups in mind | [Try it!](https://api.react-finland.fi/graphql) | [Repo](https://github.com/ReactFinland/graphql-api)
 | Shopify Storefront | The Storefront API gives you full creative control to build customized purchasing experiences for your customers. | [Try it!](https://help.shopify.com/api/storefront-graphiql) | [Docs](https://help.shopify.com/api/storefront-api)<br>[Examples](https://github.com/Shopify/storefront-api-examples)
+| SWOP | GraphQL and REST foreign exchange rate API | [Try it!](https://swop.cx/) | [Docs](https://swop.cx/documentation)
```

Comment:
@jadlr Thanks for PR 👍 
Do you have a public instance of GraphiQL or Playground to put under `Try it!`?

Reply:
Hi, we don't have a public instance. You can get a free API key. Should I remove the link?

Reply:
```suggestion
| SWOP | GraphQL and REST foreign exchange rate API. Note: Required registration. | [Try it!](https://swop.cx/) | [Docs](https://swop.cx/documentation)
```